Tasks/Common/docker-common/containerconnection.ts

```diff
@@ -73,6 +73,24 @@ export default class ContainerConnection {
         }
     }
     
+    public setDockerConfigEnvVariable() {
+        if (this.configurationDirPath && fs.existsSync(this.configurationDirPath)) {
+            tl.setVariable("DOCKER_CONFIG", this.configurationDirPath, true);
+        }
+        else {
+            tl.error(tl.loc('DockerRegistryNotFound'));
+            throw new Error(tl.loc('DockerRegistryNotFound'));
+        }
+    }
+    
+    public unsetDockerConfigEnvVariable() {
+        var dockerConfigPath = tl.getVariable("DOCKER_CONFIG");
+        if (dockerConfigPath) {
+            tl.setVariable("DOCKER_CONFIG", "");
+            del.sync(dockerConfigPath, {force: true});
```

Comment:
Should we call closeConnection here instead of del.sync.. so that certs etc can also be deleted?

Reply:
This path is to delete existing Docker_Config env directory, after this it call closeConnection call to delete current context connection.